### DISCUSSION TITLE
How to see what version of Plausible I currently run? Changelog?

### DISCUSSION THREAD

**Discussion by raabmar** [plausible/analytics#3554] [Self-Hosted Support]
Where can I see what version of Plausible I run? I followed a detailed instruction on how to set up Plausible on a small VPS and I want to test it for some time. 

However, I don't see any way to see the currently installed version.

Also, I wonder what is the official Changelog? The changelog here does not show the date/version of the latest release.
https://github.com/plausible/analytics/blob/master/CHANGELOG.md

Any idea about it? With the latest pricing changes I can't go to use Plausible as a service. I would either need to self-host or drop it.

**Comment by ruslandoga** [comment]
👋 @raabmar 

You can check the `BUILD_METADATA` env var in the `plausible/analytics` container:

```console
$ cd hosting
$ docker compose exec plausible sh -c 'echo $BUILD_METADATA'
{"tags":["plausible/analytics:v2.0.0",plausible/analytics:v2.0,plausible/analytics:v2,"plausible/analytics:latest"],"labels":{"org.opencontainers.image.created":"2023-07-12T10:25:29.191Z","org.opencontainers.image.description":"Simple, open-source, lightweight (< 1 KB) and privacy-friendly web analytics alternative to Google Analytics.","org.opencontainers.image.licenses":"AGPL-3.0","org.opencontainers.image.revision":"0621fc0a77cc50add00baddbdd59f378324da483","org.opencontainers.image.source":"https://github.com/plausible/analytics","org.opencontainers.image.title":"analytics","org.opencontainers.image.url":"https://github.com/plausible/analytics","org.opencontainers.image.version":"v2.0.0"}}
```

---

This is the log for the latest release (v2.0.0 as of now): https://github.com/plausible/analytics/blob/master/CHANGELOG.md#v200---2023-07-12

What comes after are the features that have been added but not yet released, hence titled ["Unreleased".](https://github.com/plausible/analytics/blob/master/CHANGELOG.md#unreleased)

**Reply by raabmar** [reply]
Thanks, I am glad to get a feedback here. Ok, then I got the right link for the Changelog. I don't know if I understand "Unreleased" because in my trial account (Plausible SaaS) I can see those new features, e.g. _Add searching sites by domain in /sites view_
 already

About the versioning - I'll probably need a few hours to figure out what it means. I hope I don't require many follow up questions. It seems so odd to me that it's difficult to access the version information

I thought perhaps it's easier to just update to the latest version (updating was easier than pulling the version)

For the update I just run `docker-compose pull plausible` and `docker-compose down -v && docker-compose up -d` however I think it's the same version as before and the "Unreleased" features are not included. I lost all sites and data unfortunately during the update process (it's only test data for learning as of now), but it shows me I am not really able to master all of that yet

**Reply by ruslandoga** [reply]
You probably were using `latest` image tag and updated from V1 to V2 (latest tag is tricky this way). Your data is not lost, it just needs to be migrated! Please find the discussion for V2 release and follow the guide. I'll post the links once I'm back at my computer :)

**Reply by ruslandoga** [reply]
V2 discussion: https://github.com/plausible/analytics/discussions/3132

**Reply by raabmar** [reply]
I don't think that's it. I only started a few weeks ago and v2 got out in July. I started with Plausible SaaS trial 30 days ago. It's going to be difficult to sell it to clients, so it's basically just costs for me, hence I am looking also at alternatives and self-hosted to estimate effort, risk, required competencies, learning curve, etc. but that's a bit off-topic.

I was not able to check the version number before/after the update. But basically I installed the docker image only approx. 2 weeks ago. Then I noticed the changes on the hosted Plausible trial (changelog) so I thought I give it a try to update for the first time. That's how it all started for me trying to figure out which version I am actually running on the self-hosted setup.

Too bad that my VPS didn't keep the snapshot I created 2 weeks back, would be a good chance to test if snapshot restore works.

I think I unknowingly completely replaced the entire docker image, that's why I lost all data with the update. And I also think the update did not make any difference, as I still don't see the latest features that are already available at the hosted version and documented at "[Unreleased](https://github.com/plausible/analytics/blob/master/CHANGELOG.md#unreleased)"

I can see why self-hosted is quite a challenge and maybe not worth it for everyone
